subprojects/signing/src/main/java/org/gradle/plugins/signing/Signature.java

```diff
@@ -359,8 +372,18 @@ public SignatureSpec getSignatureSpec() {
         return signatureSpec;
     }
 
+    /**
+     * Get the artifact that this signature is for, if available.
+     *
+     * @deprecated use {@link #getSource()}
```

Comment:
❌ getSource is currently private and undocumented.

Reply:
Do you think we should make `getSource()` public and document it or change the deprecation hint to "should not have been public in the first place"?

Reply:
If there is no good use case for it, then let's deprecate it without replacement for now. Please make sure to mention all deprecations in the release notes.

Reply:
Done in 4d937ebf69f.